app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -17,32 +17,84 @@ keywords:
   - teaching
   - teacher
   - special school
-quote:
-  q-jane-wilkinson:
-    text: "I specifically wanted to work with children identified with Special Educational Needs and Disabilities (SEND). I was fortunate to find a teacher training course provider that was able to allow me to practice in a mainstream school that also hosted a hearing impairment unit. And another school that offered support for children with a variety of learning needs. They were great at teaching me and letting me practice my classroom management skills early on. I was able to research and understand ways to differentiate resources for children who were on the autistic spectrum. And I was able to create and implement a scheme of work to reflect on and evaluate for later use once qualified."
-    name: "Jane Wilkinson, former teacher and teacher training adviser"
-    classes: quote--indent
+
 ---
-Most disabled pupils and pupils with special educational needs learn in mainstream schools.
+Special educational needs and disabilities (SEND) can affect pupils’ ability to learn. For example, they may struggle to read or write because they have dyslexia, or they may have difficulty concentrating. Some pupils may have physical impairments which means they need special equipment to help them learn.  
+
+Teaching pupils with SEND offers you a unique opportunity to make a lasting impact in their lives. You’ll help pupils overcome challenges and guide them to discover their strengths and fulfil their potential. Through your support, they’ll build self-esteem and resilience. 
+
+<blockquote class="quote quote--background-white quote--indent">
+  <p>I'm passionate about being there for my students, helping them see their potential and overcome their own challenges."</p>
+    <footer class="footer">
+      <div class="author">
+        <div>
+            <cite class="name">Gloria, lead maths practitioner</cite>
+        </div>
+      </div>
+</footer>
+</blockquote>
```

Comment:
@gemmadallmandfe I was struggling to get the quote component to work for this so two quotation marks showed, I I added the second quotation mark manually as a work around. Not sure if this type of thing requires a tech review?

Reply:
Not sure whether quote partials work out of the box in markdown files - worth getting some dev help on this and also looping in @emmaachesongray for info